test/test_lint.py

```diff
@@ -38,6 +39,18 @@
 ]
 
 
+def parse_err(err):
```

Comment:
Nope this all hies issues, we can't be unable to parse stuff, it should gracefully skip stuff and not print errors about encodings.

Reply:
I see. I've rebased the patch, can you please take a look?